Zinc/Private/ZincRepo.m

```diff
@@ -134,8 +134,9 @@ + (instancetype) repoWithURL:(NSURL*)fileURL networkOperationQueue:(NSOperationQ
             if (![repo queueInitializationTasks]) {
                 repo.isInitialized = YES;
             }
-            
+
             [repo queueGarbageCollectTask];
+            [repo queueContentBundleDeleteTask];
```

Comment:
actually, nevermind, there's no reason for that. forgot that this only deletes content.